airflow/operators/http_operator.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from airflow.hooks import HttpHook
+from airflow.models import BaseOperator
+from airflow.utils import apply_defaults, AirflowException
+
+
+class SimpleHttpOperator(BaseOperator):
+    """
+    Calls an endpoint on an HTTP system to execute an action
+    """
+
+    template_fields = ('endpoint',)
+    template_ext = ()
+    ui_color = '#f4a460'
+
+    @apply_defaults
+    def __init__(self,
+                 endpoint,
+                 method='POST',
+                 data=None,
+                 headers={},
```

Comment:
Instantiating objects in method headers is a python gotcha:
http://stackoverflow.com/questions/9887180/the-value-of-an-empty-list-in-function-parameter-example-here
`headers=None`
and then in the function body
```
headers = headers or {}
```